### ISSUE TITLE
[BUG] Lua Error: attempt to compare number with nil

### ISSUE THREAD

**Issue by Tga123** [Hekili/hekili#1577]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) for this issue and was not able to find one.
- [X] I edited the title of this bug report (above) so that it describes the issue I am reporting.

### Describe the Issue

```lua

18x Hekili\UI.lua:1000: attempt to compare number with nil
[string "@Hekili\UI.lua"]:1000: in function <Hekili\UI.lua:916>

Locals:
self = HekiliDisplayAOE {
	0 = <userdata>
	UpdateAlpha = <function> defined @Hekili\UI.lua:1615
	index = 4
	Active = true
	firstForce = 0
	UpdateKeybindings = <function> defined @Hekili\UI.lua:886
	Deactivate = <function> defined @Hekili\UI.lua:1865
	postRange = 0.311031
	RecommendationsStr = "vampiric_touch"
	GetPerimeterButtons = <function> defined @Hekili\UI.lua:1877
	RefreshCooldowns = <function> defined @Hekili\UI.lua:1641
	Initialized = true
	postTargets = 0.002758
	NewRecommendations = true
	delayIconShown = true
	refreshTimer = 0
	postPrimary = 0.000919
	postGlow = 0.001839
	postRecs = 0.195506
	eventCount = 297
	postAlpha = 0.001839
	updateMax = 0.555872
	updateTime = 0.228599
	auraTimer = 0
	updateCount = 477
	lastFlash = "mind_blast"
	flashTimer = 3674672.124000
	lastUpdate = 0
	eventTime = 0.003396
	postDelay = 0.174974
	UpdatePerformance = <function> defined @Hekili\UI.lua:1920
	IsThreadLocked = <function> defined @Hekili\UI.lua:908
	glowTimer = -1
	flashReady = true
	HasRecommendations = true
	numIcons = 3
	targetTimer = 0.068000
	eventMaxType = "PLAYER_TARGET_CHANGED"
	eventMax = 0.025127
	delayTimer = -1
	alphaCheck = 0.439000
	id = "AOE"
	TextureUpdateNeeded = true
	alpha = 1
	OnEvent = <function> defined @Hekili\UI.lua:1679
	recTimer = 0.139000
	SetThreadLocked = <function> defined @Hekili\UI.lua:912
	Activate = <function> defined @Hekili\UI.lua:1807
	postFlash = 0.323289
	OnUpdate = <function> defined @Hekili\UI.lua:916
	rangeTimer = -1
	threadLocked = false
}
elapsed = 0.025000
init = 3674691036.336096
Clash = 0
fixedBrightness = false
autoSnapshot = false
fixedSize = false
minimapIcon = false
screenshot = false
flashTexture = "Cooldown\star4"
builtIn = true
order = 2
rel = "CENTER"
numIcons = 3
frameLevel = 10
enabled = true
zoom = 30
keepAspectRatio = true
primaryWidth = 50
elvuiCooldown = false
frameStrata = "LOW"
y = -25
x = -335
name = "AOE"
primaryHeight = 50
postAlpha = 3674691036.337016
now = 3674691.024000
alpha = 1
```

### How to Reproduce

AOE as Venthyr Shadow Priest.

### Player Information (Link)

https://github.com/Hekili/hekili/issues

### Error Messages (Link)

https://github.com/Hekili/hekili/issues

### Additional Information

Character Data:

build: v9.2.5-1.0.7a
level: 60 (60)
class: PRIEST
spec: shadow

talents:
ancient_madness
	auspicious_spirits
	body_and_soul
	mindbender
	misery
	psychic_horror
	unfurling_darkness

covenant: venthyr

conduits:
	clear_mind = 13
	condensed_anima_sphere = 13
	haunting_apparitions = 13
	mind_devourer = 13
	rabid_shadows = 13
	translucent_image = 13

soulbinds: [Théotar_le_duc_fou]
   exquisite_ingredients = 1
   its_always_tea_time = 1
   party_favors = 1
   soothing_shade = 1
   token_of_appreciation = 1
   watch_the_shoes! = 1

sets: tier28 = 4

legendaries:
	shadow_word_manipulation = 1
	shadowflame_prism = 1

itemIDs: 173244, 173248, 182392, 188875, 188876, 188878, 188879, 188880, 189802, 189815, 189823, 189859, 190633, 192296, 192388, 69210, 89195

settings:
	aoe = 3
	buffPadding = 0
	combatRefresh = 0.1
	custom1Name = Custom 1
	custom2Name = Custom 2
	cycle = true
	cycle_min = 6
	damage = true
	damageDots = true
	damageExpiration = 6
	damagePets = true
	damageRange = 0
	debuffPadding = 0
	enabled = true
	enhancedRecheck = false
	gcdSync = true
	maxTime = 10
	nameplateRange = 8
	nameplates = false
	noFeignedCooldown = true
	package = Shadow
	petbased = false
	potion = potion_of_phantom_fire
	potionsReset = 20180919.1
	regularRefresh = 0.5
	throttleRefresh = false
	throttleTime = false
	ignore_solvent = true
	pad_ascended_blast = true
	pad_void_bolt = true
	stm_timer = 20
	sw_death_protection = 75

toggles:
	cooldowns = true [separate]
	custom1 = false
	custom2 = false
	defensives = true [separate]
	essences = true [overridden]
	interrupts = true [separate]
	mode = reactive
	potions = false

**Comment by Hekili** [comment]
I have a pending fix for this; thanks for reporting.  It does not impact the addon performance at all, it will just occur once or twice per display when they first get recommendations.

**Comment by Hekili** [comment]
Resolved in 1.0.7b.
